lib/dominoes_cases.rb

```diff
@@ -0,0 +1,23 @@
+class DominoesCase < OpenStruct
+  def test_name
+    'test_%s' % description.gsub("can't", 'can not').gsub(/[= -]+/, '_')
+  end
+
+  def workload
+    <<-WL.chomp
+input_dominoes = #{input}
+    output_chain = Dominoes.chain(input_dominoes)
+    #{can_chain ? 'assert_correct_chain(input_dominoes, output_chain)' : 'assert_nil output_chain, "There should be no chain for #{input_dominoes}"'}
```

Comment:
continuing a conversation in https://github.com/exercism/xruby/pull/484#discussion_r87055734 about whether we should have `refute_correct_chain`

Yet another argument for:

There's a bit of asymmetry in this line.

There's a slight bit more logic in `assert_nil output_chain, "There should be no chain for #{input_dominoes}"` versus `assert_correct_chain(input_dominoes, output_chain)` where all the logic is offloaded to the `example.tt` file. I think it makes more sense to be consistent, so I have added `refute_correct_chain`. If we change our minds about wanting it we just remove the commit.